roles/user_karl/tasks/main.yml

```diff
@@ -83,10 +83,14 @@
     mode: u=rw,g=r,o=r
   become: true
 
+ # Full path to `brew` is required for weird Ansible+sudo reasons: `become` resets PATH, so
+ # Homebrew may be missing. Only works if remote login user matches user_karl_target_user.
```

Comment:
> Only works if remote login user matches user_karl_target_user.
Sounds like you're saying that this `path` setting only works when the users match?